google/ggrc-core
# GGRC-6927 BE: Allow to turn off on Audit sync of people fields between Assessment and Ticket tracker issue

# Issue description

It should be possible to switch off sync of people for all Assessments on Audit.

# Steps to test the changes

1. Turn off people sync for audit;
2. Change audit captains or assessment assignees;

Expected result: no changes in reporter, assignee, verifier, ccs fields in issue tracker.

# Solution description

To implement people sync turn on/off feature the following changes where made:

1. New boolean field `people_sync_enabled` is added to `IssuetrackerIssue` model, corresponding migration is created and new alias is added to `IssueTracked`-based mixins.

2. Merge methods in assessment sync job (e.g `_merge_reporter`, `_merge_assignee`, `_merge_ccs`) are changes to take into account `people_sync_enabled` flag when merging reporter, assignee and ccs from issue tracker with data from DB.

# Sanity checklist




# PR Review checklist
